components/utilities/utilities.css

```diff
@@ -0,0 +1,50 @@
+@import url(../../css/main.css);
+.text-left{
+    text-align: left;
+}
+.text-center{
+    text-align: center;
+}
+.text-right{
+    text-align: right;
+}
+.fs-8{
+    font-size: xx-large;
+}
+.fs-7{
+    font-size: x-large;
+}
+.fs-6{
+    font-size: larger;
+}
+.fs-5{
+    font-size: large;
+}
+.fs-4{
+    font-size: medium;
+}
+.fs-3{
+    font-size: small;
+}
+.fs-2{
+    font-size: smaller;
+}
+.fs-1{
+    font-size: xx-small;
+}
+
```

Comment:
One suggestion. One line gap can be followed after class selectors or no line gap throughout the file